boot/bootutil/src/bootutil_misc.c

```diff
@@ -195,11 +195,7 @@ uint32_t
 boot_status_off(const struct flash_area *fap)
 {
     uint32_t off_from_end;
-    uint8_t elem_sz;
-
-    elem_sz = flash_area_align(fap);
-
-    off_from_end = boot_trailer_sz(elem_sz);
+    off_from_end = boot_trailer_sz(BOOT_MAX_ALIGN);
```

Comment:
This is not OK, we want to use the write size alignment for each particular MCU, this is not supposed to work on *your* MCU only.